themes/demo/assets/vendor/bootstrap/js/modal.js

```diff
@@ -63,7 +63,7 @@
     this.$body.addClass('modal-open')
 
     this.escape()
-    this.resize()
```

Comment:
@ayumihamsaki model.js has its own resize method, so not needed here as well:
https://github.com/octobercms/october/blob/fc3c4de9070f2b1b1ab95fe83390ce92d6b0a17c/themes/demo/assets/vendor/bootstrap/js/modal.js#L161

Reply:
done